polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -11,7 +11,7 @@
   --pc-index-table-loading-panel: 37;
   // stylelint-enable
   position: relative;
-  border-radius: inherit;
+  border-radius: var(--p-border-radius-2);
```

Comment:
Inheriting is unreliable when consumers need a ref between the Card component and the index table the radii get lost

Reply:
Agreed, this fix is more solid! Thanks, Kyle!